frontend/js/mixins/formStore.js

```diff
@@ -103,12 +103,5 @@ export default {
       // init value with the one present into the component itself
       this.saveIntoStore()
     }
-  },
-  beforeDestroy: function () {
```

Comment:
Just wanted to echo the comment from @ifox. With this change, if I add a block, fill some fields, then delete the block, is this content still sent to the backend on save?

Reply:
Yes I checked, but I will test it once more tomorrow to be sure.

Add/remove edit all should work fine in all languages.